src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,262 @@
+    The long history of the Web Animations API.
+  </figcaption>
+</figure>
+
+The [Web Animations API](https://www.w3.org/TR/web-animations-1/) is a tool that enables developers to write [imperative](https://www.youtube.com/watch?v=WaNoqBAp8NI) animations with JavaScript. It was written to underpin both CSS animation and transition implementations and enable future effects to be developed, as well as existing effects to be composed and timed.
+
+While it has a long history, originally launching in Chrome 36, the latest release, Chrome 84, brings us a slew of previously unsupported features. 
+
+## Getting started
+
+Creating an animation via the web animations API should feel very familiar if you've used `"@keyframe"` rules. First you'll need to create a Keyframe Object. What might look like [this](https://codepen.io/una/pen/RwWMvPw) in CSS:
+
+```css
+@keyframes openAnimation {
+  0% {
+    transform: scale(0);
+  }
+  100% {
+    transform: scale(1);
+  }
+}
+```
+
+Would look like [this](https://codepen.io/una/pen/abvYXJX) in JavaScript:
+
+```js
+const openAnimation = [
+  { transform: 'scale(0)' }, 
+  { transform: 'scale(1)' }, 
+];
+```
+
+Where you set parameters for animation in CSS:
+
+```css
+.modal {
+  animation: openAnimation 1s 1 ease-in;
+}
+```
+
+You would set in JS:
+
+```js
+document.querySelector('.modal').animate(
+    openAnimation ,  {
+    duration: 1000, // 1s
+    iterations: 1, // single iteration
+    easing: 'ease-in' // easing function
+  }
+);
+```
+
+The amount of code is about the same, but with JavaScript, you get a couple of superpowers that you don't have with CSS alone. This includes the ability to sequence effects, and an increased control of their play states.
+
+{% Aside %}
+  Hyphenated property names become camel case when used in keyframes (i.e. `"background-color"` to `"backgroundColor"`)
+{% endAside %}
+
+## Orchestrating animations with promises
+
+In Chrome 84, you now have two methods that can be used with promises: `"animation.ready"` and `"animation.finished"`.
+
+- `animation.ready` enables you to wait for pending changes to take effect (i.e. switching between playback control methods such as play and pause).
+- `animation.finished` provides a means of executing custom JavaScript code when an animation is complete.
+
+Let's continue with our example, and create an orchestrated animation chain with `"animation.finished"`. Here, you have a vertical transformation (`"scaleY"`), followed by a horizontal transformation (`"scaleX"`), followed by an opacity change on a child element: 
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-animations/modal-open.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Applying transformations and opacity to an opening modal element. <a href="https://codepen.io/una/pen/dyYKJMz">See Demo on Codepen</a>
+  </figcaption>
+</figure>
```

Comment:
Please embed this as a Glitch onto the page: https://web.dev/handbook/web-dev-components/#glitches 

[Example](https://web.dev/read-files/#read-content)

This provides a better experience because it enables readers to browse the code without leaving our site.